### ISSUE TITLE
isort behavior different on linux vs osx

### ISSUE THREAD

**Issue by robertwenquan** [timothycrosley/isort#724]
TLDR; closed. isort is fine. It's the package difference on two systems that cause the order variance. Double and triple check the packages you installed if you run into similar issue.

on MacOs, isort takes this order

$ isort -v

/#######################################################################\

     `sMMy`
     .yyyy-                                                      `
    ##soos##                                                    ./o.
          `     ``..-..`         ``...`.``         `   ```` ``-ssso```
     .s:-y-   .+osssssso/.     ./ossss+:so+:`    :+o-`/osso:+sssssssso/
     .s::y-   osss+.``.``     -ssss+-.`-ossso`   ssssso/::..::+ssss:::.
     .s::y-   /ssss+//:-.`   `ssss+     `ssss+   sssso`       :ssss`
     .s::y-   `-/+oossssso/  `ssss/      sssso   ssss/        :ssss`
     .y-/y-       ````:ssss`  ossso.    :ssss:   ssss/        :ssss.
     `/so:`    `-//::/osss+   `+ssss+-/ossso:    /sso-        `osssso/.
       \/      `-/oooo++/-      .:/++:/++/-`      ..           `://++/.

         isort your Python imports for you so you don't have to

                            VERSION 4.2.15

\########################################################################/

```
import ipywidgets as widgets
import matplotlib.pyplot as plt
from IPython.display import display
```

while on Linux, it takes a different order

$ isort -v

/#######################################################################\

     `sMMy`
     .yyyy-                                                      `
    ##soos##                                                    ./o.
          `     ``..-..`         ``...`.``         `   ```` ``-ssso```
     .s:-y-   .+osssssso/.     ./ossss+:so+:`    :+o-`/osso:+sssssssso/
     .s::y-   osss+.``.``     -ssss+-.`-ossso`   ssssso/::..::+ssss:::.
     .s::y-   /ssss+//:-.`   `ssss+     `ssss+   sssso`       :ssss`
     .s::y-   `-/+oossssso/  `ssss/      sssso   ssss/        :ssss`
     .y-/y-       ````:ssss`  ossso.    :ssss:   ssss/        :ssss.
     `/so:`    `-//::/osss+   `+ssss+-/ossso:    /sso-        `osssso/.
       \/      `-/oooo++/-      .:/++:/++/-`      ..           `://++/.

         isort your Python imports for you so you don't have to

                            VERSION 4.2.15

\########################################################################/

```
import matplotlib.pyplot as plt
from IPython.display import display
import ipywidgets as widgets
```

**Comment by robertwenquan** [comment]
on Linux 

```
$ pip freeze | grep -iE 'matplotlib|IPython|ipywidge'
ipython==5.5.0
ipython-genutils==0.2.0
matplotlib==2.0.2
```

on MacOS
```
$ pip freeze | grep -iE 'matplotlib|IPython|ipywidge'
ipython==5.7.0
ipython-genutils==0.2.0
ipywidgets==7.3.0
matplotlib==2.2.2
```

**Comment by robertwenquan** [comment]
so looks like the package versions are inconsistent across the two systems I have. 
when I create a fresh virtualenv on MacOS, with this `requirements.txt`

```
$ cat requirements.txt
ipython==5.5.0
ipython-genutils==0.2.0
matplotlib==2.0.2
```

isort gives this order
```
import matplotlib.pyplot as plt
from IPython.display import display

import ipywidgets as widgets
```

it's with the same order as it works on Linux, but with an extra line feed.

**Comment by robertwenquan** [comment]
then I installed the `ipywidgets==7.3.0` manually with `pip`
isort starts to give different result.

```
$ pip freeze | grep -iE 'matplotlib|IPython|ipywidge'
ipython==5.5.0
ipython-genutils==0.2.0
ipywidgets==7.3.0
matplotlib==2.0.2
```

isort gives below order after the `ipywidgets` is installed. 
```
import ipywidgets as widgets
import matplotlib.pyplot as plt
from IPython.display import display
```

so it looks isort is fine. The difference of the package combination lead to the different order of the import between dev and testing environment.

**Comment by robertwenquan** [comment]
after make the libraries consistent across the nodes. the output comes back to be consistent. closing
